src/components/FlatmapVuer.vue

```diff
@@ -389,9 +389,8 @@ export default {
             userData: args,
             eventType: eventType
           };
-          // Disable the nueron pop up for now.
-          if (data && data.type !== "marker")
-            this.checkAndCreatePopups(payload);
+          this.lastPayload = payload
+          this.lastData = data
```

Comment:
These two variables are not declared anywhere. I will either add them to data or declare them on mounted, and if they are declared in mounted, they should be called _lastPayload and _lastData so we know they are non-reactive members.